cli/pcluster/createami.py

```diff
@@ -111,6 +113,58 @@ def _get_cookbook_dir(region, template_url, args, tmpdir):
         sys.exit(1)
 
 
+def _is_valid_post_install_script(post_install_script_url):
+    return urlparse(post_install_script_url).scheme in ["s3", "https", "file"]
+
+
+def _get_current_timestamp():
+    return datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+
+
+def _get_post_install_script_dir(post_install_script_url, tmp_dir):
+    LOGGER.info("Post install script url: %s", post_install_script_url)
```

Comment:
Let's improve this info message in case post_install_script_url is None:

```python
LOGGER.info(
            "Post install script url %s",
            post_install_script_url if post_install_script_url else "not specified.",
        )
```

Reply:
...or we can move this message into the `if post_install_script_url:` block.

Reply:
Done